src/main/java/com/redhat/devtools/intellij/common/utils/DownloadHelper.java

```diff
@@ -150,6 +153,19 @@ public static DownloadHelper getInstance() {
         return result;
     }
 
+    private ToolsConfig.Platform getPlatformBasedOnOs(ToolsConfig.Tool tool) {
+        String osArch = System.getProperty("os.arch");
+        String osId = Platform.os().id();
+        if (osArch != null
+            && tool.getPlatforms().containsKey(osId + "-" + osArch)) {
+                return tool.getPlatforms().get(osId + "-" + osArch);
+        } else if ("arm64".equals(osArch)) {
```

Comment:
I don't get this code. Ex MacOS M1 is supposed to support Intel binaries

Reply:
Ah i thought it only worked with arm binaries. 
Found it "MacOS contains Rosetta 2 software that does dynamic binary translation from x86, so that x86 software can be run on the M1 CPU. Not quite as efficient as code compiled directly from C to AArch64 machine code, but it works."